### DISCUSSION TITLE
Will the 0.6.1 version be published to Typst Universe?

### DISCUSSION THREAD

**Discussion by ironupiwada** [RolfBremer/in-dexter#23] [General]
I've noticed that the last version of in-dexter is 0.6.1, whereas the last version uploaded to Typst Universe remains 0.5.3. Will be there any chance to upgrade the package (presumably in accordance with Typst 0.12)?

**Comment by RolfBremer** [comment]
Yes, we wanted to implement another detail first, but have to delay it a bit. So we will submit the current version to the Typst Universe during the next days.
